assets/i18n/qqq.json

```diff
@@ -104,6 +104,7 @@
 	"I18N_DASHBOARD_TABLE_HEADING_PLAYS": "Text shown as table header in list view for explorations on the creator's dashboard - This is the table header for the Plays column.\n{{Identical|Play}}",
 	"I18N_DASHBOARD_TABLE_HEADING_RATING": "Text shown as table header in list view for explorations on the creator's dashboard - This is the table header for the Rating column.\n{{Identical|Rating}}",
 	"I18N_DASHBOARD_TABLE_HEADING_UNRESOLVED_ANSWERS": "Text shown as table header in list view for explorations on the creator's dashboard - This is the table header for the Unresolved Answers column.",
+	"I18N_DEFAULT_MOBILE_PLACEHOLDER": "Text shown as the default placeholder for learners using mobile devices - this text is shown only if no placeholder is specified by the creator.",
```

Comment:
Can you change the name to I18N_PLAYER_DEFAULT_MOBILE_PLACEHOLDER, just so that it's grouped with the other player strings?

And can you use this text: "Text displayed as the default placeholder on mobile devices when no custom placeholder is specified by the creator - This text is shown inside the text-box where learners input their answer to indicate that the text-box must be tapped first to start typing an answer on mobile devices."

Reply:
Done.